src/components/Synthetics/ConfirmationBox/ConfirmationBox.tsx

```diff
@@ -1044,6 +1043,8 @@ export function ConfirmationBox(p: Props) {
         ? applySlippageToPrice(allowedSlippage, increaseAmounts.acceptablePrice, true, isLong)
         : increaseAmounts?.acceptablePrice;
 
+    const collateralSpreadPercent = collateralSpreadInfo?.spread?.mul(10000)?.div(expandDecimals(1, USD_DECIMALS));
```

Comment:
You can use the `BASIS_POINTS_DIVISOR` form config/factors.ts instead of using 10000

Reply:
Good idea, thank you